### PR TITLE
added the option to manually collect rewards

### PR DESCRIPTION
**Pull request by quendera**
As discussed on #47 , here is a simple implementation to manually collect rewards.

I made the default to be automatic collection and made the last action of the agent to be collect - adaptable to use_noop.

My tests seemed to have no issues, but please do carefully review.

### CODE REVIEW

**Comment by quendera** [comment]
i just realised I need to fix a feature before PR. Please hold before merging

File: neuronav/envs/grid_env.py [hunk 1]
```diff
@@ -755,35 +760,60 @@ def step(self, action: int):
         if self.stochasticity > self.rng.rand():
             action = self.rng.randint(0, self.action_space.n)
 
+        if self.manual_collect == False:
```
Comment (awjuliani) [inline neuronav/envs/grid_env.py:763]: Shouldn't this be reversed? If manual collect is false, then collection should be automatic?

Comment (quendera) [reply neuronav/envs/grid_env.py:763]: you are right, it came as a result of me changing the naming convention from auto_collect to manual_collect, which reverses the logic.

**Comment by awjuliani** [comment]
Looks good to me. Thanks for implementing this! I've created a new PR here with this feature along with and additional test and version number increment. https://github.com/awjuliani/neuro-nav/pull/49

**Pull request closed without merging**